data_structures/trie_beginner.cpp

```diff
@@ -0,0 +1,153 @@
+/**
+ * @file
+ * @brief https://en.wikipedia.org/wiki/Trie
```

Comment:
Preferably, use Markdown links.
```suggestion
 * @brief [Beginner Trie](https://en.wikipedia.org/wiki/Trie) in data structures
 *
```